setup-production.adoc

```diff
@@ -347,10 +372,37 @@ mv settings/config.py.prod.example settings/config.py
 
 Edit it with your own configuration:
 
-- connection to PostgreSQL
-- connection to RabbitMQ for `taiga-events` and `taiga-async`
-- credentials for email
-- Enable/disable anonymous telemetry
+- Taiga secret key: **it's important** to change it. It must have the same value as the secret key in `taiga-events` and `taiga-protected`
+- Taiga urls: configure where Taiga would be served using `TAIGA_URL`, `SITES` and `FORCE_SCRIPT_NAME` (see examples below)
+- Connection to PostgreSQL; check `DATABASES` section
+- Connection to RabbitMQ for `taiga-events`; check "EVENTS" section
+- Connection to RabbitMQ for `taiga-async`; check "TAIGA ASYNC" section
```

Comment:
Same as above, but in this case it's minor because taiga-async doesn't exist in the documentation index.